### DISCUSSION TITLE
Better Module / Theme installment.

### DISCUSSION THREAD

**Discussion by PossiblePanda** [NamelessMC/Nameless#2359] [Ideas]
I think it would be a really good idea to be able to download themes/modules online, instead of having to edit files. This can be useful for people that might have issues with importing the files.

**Comment by tadhgboyle** [comment]
Hey, thanks for suggestion. 
Currently you download a module online in a zip file, and then upload the files to namelessmc. 

I am confused what you mean - do you want a one click download and install button on the panel? Because we already have what you have described.
